libs/core/src/lib/tabs/tab-item/tab-item.directive.ts

```diff
@@ -18,12 +18,9 @@ export type TabItemState = 'success' | 'error' | 'warning' | 'information' | 'ne
 @Directive({
     // tslint:disable-next-line:directive-selector
     selector: '[fd-tab-item]',
-    host: {
-        class: 'fd-tabs__item'
-    }
 })
 export class TabItemDirective implements CssClassBuilder, OnChanges, OnInit {
-    /** Apply user custom styles */
+    /** @hidden Apply user custom styles */
```

Comment:
Why do you hide it? If it's hidden it should be prefixed with `_`

Reply:
Change reverted